dbt-snowflake/src/dbt/adapters/snowflake/relation.py

```diff
@@ -151,75 +142,13 @@ def can_be_renamed(self) -> bool:
         """
         return self.type in self.renameable_relations and not self.is_iceberg_format
 
-    def get_ddl_prefix_for_create(self, config: RelationConfig, temporary: bool) -> str:
```

Comment:
if we remove this won't we break anyone who has custom macros that depend on this? We may want to leave it for backwards compatibility

Reply:
Oh, I suppose.

Reply:
I'll add this and `get_iceberg_ddl_options` back in, though they may only work as expected when the user explicitly sets the `table_format`. I think I updated the code in a way where that's actually not the case, but I can't guarantee it.